xnat_tools/bids_postprocess.py

```diff
@@ -1,131 +1,84 @@
-import sys
 import os
-import argparse
-import coloredlogs
+import typer
 import logging
+from typing import List
 from xnat_tools.bids_utils import insert_intended_for_fmap
+from xnat_tools.logging import setup_logging
 
 _logger = logging.getLogger(__name__)
 
-
-def parse_args(args):
-    """Parse command line parameters
-
-    Args:
-      args ([str]): command line parameters as list of strings
-
-    Returns:
-      :obj:`argparse.Namespace`: command line parameters namespace
+app = typer.Typer()
+
+
+@app.command()
+def bids_postprocess(
+    session: str = typer.Argument(
+        ..., help="XNAT Session ID, that is the Accession # for an experiment."
+    ),
+    bids_experiment_dir: str = typer.Argument(
+        ..., help="Root output directory for exporting the files"
+    ),
+    session_suffix: str = typer.Option(
+        "01",
+        "-ss",
+        "--session-suffix",
+        help="Suffix of the session for BIDS defaults to 01. This will produce a session label of sess-01. You likely only need to change the dault for multi-session studies",
+    ),
+    includesubj: List[str] = typer.Option(
+        [],
+        "-i",
+        "--includeseq",
+        help="Include this participant only, can be specified multiple times",
+    ),
+    skipsubj: List[str] = typer.Option(
+        [],
+        "-s",
+        "--skipseq",
+        help="Skip this participant, can be specified multiple times",
+    ),
+    log_file: str = typer.Option("", help="File to senf logs to",),
+    verbose: bool = typer.Option(
+        False, "-v", help="Verbose logging. If True, sets loglevel to INFO"
+    ),
+    very_verbose: bool = typer.Option(
+        False, "--vv", help="Very verbose logging. If True, sets loglevel to DEBUG"
```

Comment:
I think we should consider `-vv` via the `count` option from `typer`. This will make our flags more similar to many other unix tools

https://typer.tiangolo.com/tutorial/parameter-types/number/#counter-cli-options